homeassistant/components/graphite/__init__.py

```diff
@@ -31,6 +33,9 @@
             {
                 vol.Optional(CONF_HOST, default=DEFAULT_HOST): cv.string,
                 vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.port,
+                vol.Optional(CONF_PROTOCOL, default=DEFAULT_PROTOCOL): vol.Any(
+                    "tcp", "udp"
```

Comment:
Let's define and use some integration level constants for these:

```suggestion
                    PROTOCOL_TCP, PROTOCOL_UDP
```

Reply:
Can't find such constants in core. Do you want me to define them?

Reply:
Yes, they can be defined right in this file here.